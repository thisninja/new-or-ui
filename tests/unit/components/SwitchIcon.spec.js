import Component from '@/components/SwitchIcon/main.vue';

test('renders correctly', () => {
    const { element } = shallowMount(Component);
    expect(element).toMatchSnapshot();
});
