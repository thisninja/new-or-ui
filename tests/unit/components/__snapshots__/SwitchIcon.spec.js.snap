// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="cl-switch-icon clickable"
  style="width: 48px; height: 28px; background-color: rgb(245, 248, 250); justify-content: flex-end; border-radius: 14px; padding: 4px;"
>
  <cl-icon />
</div>
`;
